### ISSUE TITLE
Mongoid_rails_migrations is incompatible with Mongoid versions 4.0.0

### ISSUE THREAD

**Issue by regis-leray** [adacosta/mongoid_rails_migrations#31]
rails version 4.1.6
mongoid 4.0.0
ruby-2.1.1

```
rails generate mongoid:migration convert_location_to_venue
```

```
/Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/mongoid_rails_migrations-1.0.1/lib/rails/generators/mongoid/mongoid_generator.rb:17:in `next_migration_number': undefined method `timestamped_migrations' for Mongoid::Config:Module (NoMethodError)
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/generators/migration.rb:41:in `set_migration_assigns!'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/generators/migration.rb:57:in `migration_template'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/mongoid_rails_migrations-1.0.1/lib/rails/generators/mongoid/migration/migration_generator.rb:8:in `create_migration_file'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/generators.rb:157:in `invoke'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/commands/generate.rb:11:in `<top (required)>'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `block in require'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:135:in `generate_or_destroy'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:51:in `generate'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/regis.leray/.rvm/gems/ruby-2.1.1@global/gems/railties-4.1.6/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

**Comment by adacosta** [comment]
If you or someone else would like to make the fix, I'll do the merge.

**Comment by arthurnn** [comment]
Will look into a fix for this.

**Comment by adacosta** [comment]
I pushed out a fix which will be for 1.1.0, currently in master. I haven't had a chance to test if it works in a rails project. I've configured dependencies for mongoid >= 4.0.0 and rails >= 4.2.0. If you get a chance, let me know if it works, otherwise I'll try to check it tonight.

**Comment by regis-leray** [comment]
@adacosta  thanks it work now. tested with master
